packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -221,3 +225,60 @@ export interface ISummarizer
         options: Omit<ISubmitSummaryOptions, "summaryLogger">,
     ): OnDemandSummarizeResult;
 }
+
+/**
+ * Data about a summary attempt
+ */
+ export interface ISummaryAttempt {
+    /**
+     * Reference sequence number when summary was generated or attempted
+     */
+    readonly refSequenceNumber: number;
+
+    /**
+     * Time of summary attempt after it was sent or attempted
+     */
+    readonly summaryTime: number;
+
+    /**
+     * Sequence number of summary op
+     */
+    summarySequenceNumber?: number;
+}
+
+export interface ISummarizerHeuristics {
+    /** Last received op sequence number */
+    lastOpSequenceNumber: number;
+
+    /** Last sent summary attempt */
+    readonly lastAttempt: ISummaryAttempt;
+
+    /** Last summary attempt that received an ack */
+    readonly lastAck: ISummaryAttempt;
```

Comment:
Consider calling this `lastAckedAttempt` and `lastAttempt` -> `lastSentAttempt`. Currently, the names are a little confusing :-).

Reply:
How about `lastSuccessfulSummary`?